packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/livestream/host-livestream.mdx

```diff
@@ -0,0 +1,117 @@
+---
+id: host-livestream
+title: HostLivestream
+---
+
+import ImageShowcase from '@site/src/components/ImageShowcase';
+import HostLivestreamStart from '../../assets/04-ui-components/livestream/host-livestream-start.png';
+import HostLivestreamEnd from '../../assets/04-ui-components/livestream/host-livestream-end.png';
+
+import HostLivestreamTopView from '../../common-content/ui-components/livestream/host-livestream/host-livestream-top-view.mdx';
+import LivestreamLayout from '../../common-content/ui-components/livestream/livestream-layout.mdx';
+import HostLivestreamControls from '../../common-content/ui-components/livestream/host-livestream/host-livestream-controls.mdx';
+import DurationBadge from '../../common-content/ui-components/livestream/duration-badge.mdx';
+import FollowerCount from '../../common-content/ui-components/livestream/follower-count.mdx';
+import LiveIndicator from '../../common-content/ui-components/livestream/live-indicator.mdx';
+import HostStartStreamButton from '../../common-content/ui-components/livestream/host-livestream/host-start-stream-button.mdx';
+import LiveStreamMediaControls from '../../common-content/ui-components/livestream/livestream-media-controls.mdx';
+import OnStartStreamHandler from '../../common-content/ui-components/livestream/host-livestream/on-start-stream-handler.mdx';
+import OnEndStreamHandler from '../../common-content/ui-components/livestream/host-livestream/on-end-stream-handler.mdx';
+
+The `HostLivestream` is the UI component that allows you to show the live stream view in the app on the host's end with minimal efforts. So you don't need to take care much about each feature that you need to build a video live stream call screen with this component.
```

Comment:
```suggestion
The `HostLivestream` is the UI component that allows you to show the livestream view in the app on the host's end with minimal efforts. So you don't need to take care much about each feature that you need to build a video livestream call screen with this component.
```

Reply:
livestream is a single word afaik. Please correct the same in all other instances as well, unless you disagree :)

Reply:
Done ✅